test/test.awsclient.ts

```diff
@@ -203,6 +203,63 @@ describe('AwsClient', () => {
       });
     });
 
+    it('should throw when an unsupported url is provided', () => {
```

Comment:
Missing tests for fd00:ec2::254 I think. Might as well explicitly test w/ the set of allowed hosts.